src/base/memory_monitor_sr.cpp

```diff
@@ -393,6 +537,7 @@ namespace cubperf
 /* APIs */
 int mmon_initialize (const char *server_name)
 {
+  int error = NO_ERROR;
   cubperf::mmon_Gl = new cubperf::memory_monitor (server_name);
 
   if (cubperf::mmon_Gl == nullptr)
```

Comment:
if you would use variable `error`, then it would be better to set error here. 
`error = ER_...`

Reply:
Done.